apps/fem/fem2.cpp

```diff
@@ -0,0 +1,139 @@
+#include "graph.h"
```

Comment:
I think the name I would like is `fem_scalar_equations_linear_triangle_discretization_2D.cpp` :stuck_out_tongue_closed_eyes:

All jokes aside, one has to find the largest set of problems this can be applied to. Currently this is all scalar equations using linear triangle basis functions. The next thing would then be to solve the [helmholtz equation](https://en.wikipedia.org/wiki/Helmholtz_equation) which could use the same `*.cpp` file, but a different `*.sim` file. One can **not** solve the [biharmonic equation](https://en.wikipedia.org/wiki/Biharmonic_equation) since the linear triangles are insufficient and one can **not** solve [(non-)linear elasticity](https://en.wikipedia.org/wiki/Linear_elasticity) since this has 2 unknowns (displacement in x- and y-direction) per spatial point, i.e. a vector equation.

So `triangle_scalar_2D.cpp` could be a nice one. Omitting `fem` since we're in that folder already.